types/node/events.d.ts

```diff
@@ -58,6 +58,213 @@ declare module "events" {
+                /**
+                 * The millisecond timestamp when the `Event` object was created.
+                 */
+                readonly timeStamp: number;
+                /**
+                 * The event type identifier.
+                 */
+                readonly type: string;
+
+                static readonly NONE: 0;
+                static readonly CAPTURING_PHASE: 1;
+                static readonly AT_TARGET: 2;
+                static readonly BUBBLING_PHASE: 3;
+            }
+
+            interface EventLike {
+                type: string;
+            }
+
+            interface EventListenerOptions {
+                /**
+                 * Not directly used by Node.js. Added for API completeness.
+                 */
+                capture?: boolean;
+            }
+
+            interface AddEventListenerOptions extends EventListenerOptions {
+                /**
+                 * When `true`, the listener is automatically removed when it is first invoked
+                 */
+                once?: boolean;
+                /**
+                 * When `true`, serves as a hint that the listener will not call
+                 * the `Event object's `preventDefault()` method.
+                 */
+                passive?: boolean;
+            }
+
+            type EventListener = ((event: Event) => void) | { handleEvent(event: Event): void; };
+
+            class EventTarget {
+                constructor();
+                /**
+                 * Adds a new handler for the `type` event. Any given `listener` is added
+                 * only once per `type` and per `capture` option value.
+                 *
+                 * If the `once` option is `true`, the `listener` is removed after the next time a `type` event is dispatched.
+                 *
+                 * The `capture` option is not used by Node.js in any functional way other than
+                 * tracking registered event listeners per the `EventTarget` specification.
+                 * Specifically, the `capture` option is used as part of the key when registering a `listener`.
+                 * Any individual `listener` may be added once with `capture = false`, and once with `capture = true`.
+                 */
+                addEventListener(type: string, listener: EventListener, options?: boolean | AddEventListenerOptions): void;
+                /**
+                 * Dispatches the `event` to the list of handlers for `event.type`.
+                 *
+                 * The registered event listeners is synchronously invoked in the order they were registered.
+                 */
+                dispatchEvent(event: Event | EventLike): boolean;
+                /**
+                 * Removes the `listener` from the list of handlers for event `type`.
+                 */
+                removeEventListener(type: string, callback: EventListener, options?: EventListenerOptions | boolean): void;
+            }
```

Comment:
These need to be moved to the top‑level of `global`, since otherwise they won’t be accessible through the identifiers or `globalThis`, and the `NodeJS` namespace doesn’t exist at runtime.